src/assets/Components/ArtistName/ArtistName.jsx

```diff
@@ -0,0 +1,33 @@
+export const ArtistName = ({ artists }) => {
+  const displayArtists = () => {
+    const artistLength = artists.artistLength
+    if (artistLength === 0) return ''
+    // if only one print the name
+    if (artistLength === 1) return artists[0].name
+    // take the last 2 artists names
+    const lastTwoArtists = artists.slice(-2).map((artist) => artist.name)
+    // take the other artists which are not the last two and add comma in between
+    const otherArtists = artists
+      .slice(0, -2)
+      .map((artist) => artist.name)
+      .join(',')
+    // finally print only a comma for the artists before the last 2 and and & between last 2
+    return `${
+      otherArtists.length ? otherArtists + ',' : ''
+    }${lastTwoArtists.join(' & ')}`
+  }
```

Comment:
I don't completely follow this code to be honest 👀 And it seems to mess up the links:

![Skärmavbild 2024-03-13 kl  11 03 45](https://github.com/Technigo/project-music-releases-vite/assets/51529201/342b04ce-9ee1-40ba-8b48-124bbb8ce186)